build/NI.CSharp.Analyzers.nuspec

```diff
@@ -27,7 +27,7 @@
     </dependencies>
   </metadata>
   <files>
-    <file src="..\.binaries\Release\NationalInstruments.Analyzers\**\*.dll" target="analyzers\dotnet\cs" />
+    <file src="..\.binaries\Release\NationalInstruments.Analyzers\netstandard1.3\*.dll" target="analyzers\dotnet\cs" />
```

Comment:
I don't care this was changed, but was it necessary? Just trying to understand

Reply:
I just align it to the same structure as the one from Microsoft Analyzer. As mentioned above, the dlls will be directly put under the dotnet\cs folder instead of dotnet\cs\netstandard1.3